### PR TITLE
Topology bug fix

### PR DESCRIPTION
**Pull request by luoyuctl**
#### What type of this PR
Topology bug fix

Add one of the following kinds:
/kind bug

#### Which issue(s) this PR fixes:
Fixes #
- Solve the problem of overcalculating the number of application instances
- Fixed an issue where the running count of application instances on the topology page was incorrectly displayed 
- Fixed the incorrect number of incorrect requests and service exceptions in the global overview

### CODE REVIEW

File: modules/monitor/apm/topology/topology.go [hunk 1]
```diff
@@ -1358,6 +1356,7 @@ func (topology *provider) Services(serviceName string, nodes []*Node) []ServiceD
 
 type ServiceInstance struct {
 	ApplicationName     string `json:"applicationName,omitempty"`
+	ServiceId           string `json:"serviceId,omitempty"`
```
Comment (luobily) [inline modules/monitor/apm/topology/topology.go:1359]: Proper nouns need to be capitalized, e.g. ID

**Review by luobily — CHANGES_REQUESTED** [review]

File: modules/monitor/apm/topology/topology.go (hunk 1 above)
Comment (luoyuctl) [reply modules/monitor/apm/topology/topology.go:1359]: Combining id with other fields is universal, and id as a separate field needs to be all uppercase

Comment (luoyuctl) [reply modules/monitor/apm/topology/topology.go:1359]: > Proper nouns need to be capitalized, e.g. ID
Combining id with other fields is universal, and id as a separate field needs to be all uppercase

Comment (luoyuctl) [reply modules/monitor/apm/topology/topology.go:1359]: > Proper nouns need to be capitalized, e.g. ID

Combining id with other fields is universal, and id as a separate field needs to be all uppercase

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**